modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -233,6 +245,34 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return this._network;
   }
 
+  static blockExplorer: UTXOBlockExplorer = {
```

Comment:
static shouldn't be in an abstract class - please add this to a concrete implementation or break it out in its own file. in general all this logic should not live in abstract classes.